graphql_compiler/compiler/helpers.py

```diff
@@ -315,7 +315,7 @@ def get_parameter_name(argument: str) -> str:
 LocationT = TypeVar("LocationT", bound="BaseLocation")
 
 
-@total_ordering
+@total_ordering  # type: ignore
```

Comment:
Maybe link to the mypy issue, so we know why this is ignored and when it might be safe to remove the override?